modules/oe_starter_content_person/oe_starter_content_person.module

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @file
+ * Main *.module file.
```

Comment:
I always write "Main *.module file" because there is nothing to learn from this comment, and by omitting the module name it can be copied anywhere.
I mean we already know which module this is, no point in repeating it here.
But I can follow a convention if need be.

Reply:
Looking into oe_content, I find the following patterns for `@file` comment texts in *.module files:
- 'Primary module hooks for OpenEuropa Call For Proposals content module.'
- 'The module file.'
- 'The OpenEuropa Content Event module.'
- 'OE Content Organisation module file.'
- 'The OpenEuropa Content Sub Entity Author.'

I prefer "The module file." because it is the only one that can be copied without the need to adjust.